release-notes/supported-os.json

```diff
@@ -0,0 +1,121 @@
+                    "eos-date": "2023-05-31"
+                },
+                {
+                    "cycle": "20.04",
+                    "supported-channels": ["6.0", "7.0", "8.0"],
+                    "unsupported-channels": [],
+                    "eos-date": "2022-10-01"
+                },
+                {
+                    "cycle": "22.04",
+                    "supported-channels": ["6.0", "7.0", "8.0", "9.0"],
+                    "unsupported-channels": [],
+                    "eos-date": "2024-09-30"
+                },
+                {
+                    "cycle": "22.10",
+                    "supported-channels": [],
+                    "unsupported-channels": ["6.0", "7.0"],
+                    "eos-date": "2023-07-20"
+                },
+                {
+                    "cycle": "23.04",
+                    "supported-channels": [],
+                    "unsupported-channels": ["6.0", "7.0"],
+                    "eos-date": "2024-01-20"
+                },
+                {
+                    "cycle": "23.10",
+                    "supported-channels": ["6.0", "7.0", "8.0"],
+                    "unsupported-channels": [],
+                    "eos-date": "2024-07-01"
+                },
+                {
+                    "cycle": "24.04",
+                    "supported-channels": ["6.0", "7.0", "8.0", "9.0"],
+                    "unsupported-channels": [],
+                    "eos-date": "2029-07-01"
+                }
+            ]
+        },
+        {
+            "name": "iOS",
+            "lifecycle-policy": "https://support.apple.com/ios",
+            "last-updated": "2023-02-28",
+            "upcoming-eos": {
+                "cycle": "15",
+                "expected-eos-date": "2024-10-01"
+            },
+            "cycles": [
+                {
+                    "cycle": "10",
+                    "supported-channels": [],
+                    "unsupported-channels": ["6.0", "7.0"],
+                    "eos-date": "2017-09-26"
+                },
+                {
+                    "cycle": "11",
+                    "supported-channels": [],
+                    "unsupported-channels": ["6.0", "7.0", "8.0"],
+                    "eos-date": "2018-10-08"
+                },
+                {
+                    "cycle": "12",
+                    "supported-channels": [],
+                    "unsupported-channels": ["6.0", "7.0", "8.0"],
+                    "eos-date": "2023-01-23"
+                },
+                {
+                    "cycle": "13",
+                    "supported-channels": [],
+                    "unsupported-channels": ["6.0", "7.0", "8.0"],
+                    "eos-date": "2020-09-16"
+                },
+                {
+                    "cycle": "14",
+                    "supported-channels": [],
+                    "unsupported-channels": ["6.0", "7.0", "8.0"],
+                    "eos-date": "2021-10-01"
```

Comment:
For iOS, we should take `eol` instead of `support` as base from [eol.date](https://endoflife.date/ios), since `support` means "Apple gives technical support" and not "Apple offers security updates" in this case. As of now, Microsoft still supports the cycles that receive security updates to my understanding.

This is how those look at EOL.API.
```
  {
    "cycle": "15",
    "releaseDate": "2021-09-20",
    "support": "2022-09-12",
    "eol": false,
    "latest": "15.8.1",
    "latestReleaseDate": "2024-01-22",
    "link": "https://support.apple.com/HT212788",
    "lts": false
  },
  {
    "cycle": "14",
    "releaseDate": "2020-09-16",
    "support": "2021-09-20",
    "eol": "2021-10-01",
    "latest": "14.8.1",
    "latestReleaseDate": "2021-10-26",
    "link": "https://support.apple.com/HT211808",
    "lts": false
  }
```